worker/src/main/scala/org/apache/celeborn/service/deploy/worker/Worker.scala

```diff
@@ -592,7 +603,42 @@ private[celeborn] class Worker(
       logWarning(s"Waiting for all PartitionLocation release cost ${waitTime}ms, " +
         s"unreleased PartitionLocation: \n$partitionLocationInfo")
     }
-    stop(CelebornExitKind.WORKER_GRACEFUL_SHUTDOWN)
+  }
+
+  def decommissionWorker(): Unit = {
+    try {
+      masterClient.askSync[PbWorkerLostResponse](
```

Comment:
Report WorkerLost is useless while heartbeat still report worker alive to master, probably we need a new decommission(or may use shutdown status, I think it's ok) status to represent current worker state.

Reply:
I have checked that if we sent WorkerLost, this worker will be removed from `statusSystem.workers`, then when heartbeat, it won't be removed from `excludedWorkers`, only will be added back after a `RegisterWorker`

Reply:
Yea, should also use shutdown status, it's same meaning